src/parkapi_sources/converters/radvis_bw/converter.py

```diff
@@ -38,7 +38,7 @@ def get_static_parking_sites(self) -> tuple[list[StaticParkingSiteInput], list[I
         static_parking_site_errors: list[ImportParkingSiteException] = []
 
         parking_site_features = self.geojson_validator.validate(self.get_data())
-        sources_to_ignore: list[str] = []
+        sources_to_ignore: list[str] = ['MOBIDATABW']
```

Comment:
That should be part of the config as loaded below, not hardcoded in this converter.

Reply:
Now set in the unit test.